python/desc/sims/GCRCatSimInterface/DatabaseEmulator.py

```diff
@@ -157,8 +189,32 @@ def _init_data_indices(self):
             except (TypeError, IndexError):
                 radius_rad = self._obs_metadata._boundLength
 
-            ra = self._descqa_obj._catalog['raJ2000']
-            dec = self._descqa_obj._catalog['decJ2000']
+            if 'healpix_pixels' in dir(descqa_catalog):
```

Comment:
Maybe do instead:
```python
if 'healpix_pixel' in descqa_catalog._native_filter_quantities:
```
The fact that `healpix_pixels` is in `descqa_catalog` is kind of implementation detail that may not persist.